pytest/tests/loadtest/locust/common/ft.py

```diff
@@ -75,3 +124,55 @@ def sign_and_serialize(self, block_hash):
                                                  int(3E14), int(1E23),
                                                  account.use_nonce(),
                                                  block_hash)
+
+
+@events.init.add_listener
+def on_locust_init(environment, **kwargs):
+    # Note: These setup requests are not tracked by locust because we use our own http session
+    host, port = environment.host.split(":")
+    node = cluster.RpcNode(host, port)
+
+    ft_contract_code = environment.parsed_options.fungible_token_wasm
+    num_ft_contracts = environment.parsed_options.num_ft_contracts
+    funding_account = NearUser.funding_account
+    parent_id = funding_account.key.account_id
+    worker_id = "local_id"
+    if hasattr(environment.runner, "worker_id"):
```

Comment:
You can slighly simplify this construct using the additional argument of `getattr (object, name[, default])` [1] 
```python
worker_id = getattr(environment.runner, "worker_id", "local_id")
```

[1] https://python-reference.readthedocs.io/en/latest/docs/functions/getattr.html

Reply:
done